core/metrics/src/main/scala/org/typelevel/otel4s/metrics/ObservableCounter.scala

```diff
@@ -92,6 +95,40 @@ object ObservableCounter {
       *   callbacks are ignored.
       */
     def createObserver: F[ObservableMeasurement[F, A]]
+
+    /** Modify the context `F` using an implicit [[KindTransformer]] from `F` to `G`.
+      */
+    def mapK[G[_]: MonadCancelThrow](implicit
```

Comment:
I think when both `F ~> G` and `G ~> F` are required, the method is more typically named `imapK`, e.g. [in `InvariantK` from cats-tagless](https://github.com/typelevel/cats-tagless/blob/012c6e1ff92974c0d8ad13dfa4c64ec1a741485e/core/src/main/scala/cats/tagless/InvariantK.scala#L27-L29).